jax/_src/debugger/cli_debugger.py

```diff
@@ -95,7 +95,7 @@ def do_p(self, arg):
       self._error_message()
 
   def do_pp(self, arg):
-    """pp expression
+    """Pp expression
```

Comment:
The two changes in this file seem wrong because `p` and `pp` here are case-sensitive. We could change them to `Do a p expression` to satisfy the "start with a capital letter" lint.